src/palace/manager/util/http.py

```diff
@@ -230,39 +286,24 @@ def _request_with_timeout(
         )
         backoff_factor: float = float(kwargs.pop("backoff_factor", 1.0))
 
-        # Unicode data can't be sent over the wire. Convert it to UTF-8.
-        if "data" in kwargs and isinstance(kwargs["data"], str):
-            kwdata: str = kwargs["data"]
-            kwargs["data"] = kwdata.encode("utf8")
-
```

Comment:
This may have been true at one point in python 2, but its not in python 3. Requests is happy to accept `str`.